app/renderer/js/utils/reconnect-util.js

```diff
@@ -1,42 +1,63 @@
-const isOnline = require('is-online');
+const { ipcRenderer } = require('electron');
+const request = require('request');
+const backoff = require('backoff');
+
 const Logger = require('./logger-util');
+const RequestUtil = require('./request-util');
+const DomainUtil = require('./domain-util');
 
 const logger = new Logger({
 	file: `domain-util.log`,
 	timestamp: true
 });
 
 class ReconnectUtil {
-	constructor(serverManagerView) {
-		this.serverManagerView = serverManagerView;
-		this.alreadyReloaded = false;
+	constructor(webview) {
+		this.webview = webview;
+		this.url = webview.props.url;
+		this.exponentialBackoff = backoff.exponential();
```

Comment:
The default backoff goes to a maximum of 10s., which is a little too frequent in my opinion.
I'd start with 5 seconds and the fibonacci strategy:
```suggestion
		this.exponentialBackoff = backoff.fibonacci({
                  initialDelay: 5000,
                  maxDelay: 300000
                });
```
that would generate 5s, 5s, 10s, 20s, 30s, 50s, 80s, 130s, 210s, 300s, 300s...

Reply:
> The default backoff goes to a maximum of 10s., which is a little too frequent in my opinion.

I agree; I'll change that. 

> the fibonacci strategy:

Definitely not a problem, but I'm curious to know why you think that's better than exponential.

Reply:
It grows a little bit slower.
5s, 10s, 20s, 40s, 80s, 160s, 300s would be the exponential one with factor 2.

Reply:
Sure, makes sense. I'll push the changes right away.